pages/leader_board.html

```diff
@@ -1,7 +1,7 @@
 <div ng-controller="leaderboard as lb">
     <div class="league_landing">
         <div class="league_button_history">
-            <img src="images/Trophy.png"></img> Leader Board
+            <img src="../images/Trophy.png"></img> Leader Board
```

Comment:
green one should be correct with the leaderboard file in the pages folder, you have to go up 1 directory into the images folder to get to trophy.png